lib/util.js

```diff
@@ -2,34 +2,63 @@ const unicode = require('../lib/unicode')
 
 module.exports = {
     isSpaceSeparator (c) {
```

Comment:
`unicode.Space_Separator` is no longer referenced by this function, so it can be removed from `lib/unicode.js` by modifying `build/unicode.js` and updating `lib/unicode.d.ts` accordingly.

This is technically a breaking change, so it might be better to wait for json5@3 before removing `unicode.Space_Separator`.

Reply:
I'll leave it in then.